docs/en/enterprise-edition/content-collections/dashboards/create-and-manage-dashboards.adoc

```diff
@@ -83,7 +83,7 @@ Note: If an alert is opened and resolved more than once a day, the latest resolu
 |*Compliance Trend*
 |Compliance posture trend over time.
 
-Note: If you select *Time Range > Custom*, only the Start Date day will be applied to the widget, not the hourly timestamp. In addition, the End date will always be the current date. 
+Note: If you select *Time Range > Custom*, only the Start Date day will be applied to the widget, not the hourly timestamp. In addition, you cannot alter the End date, which is set by default to the current date. 
```

Comment:
I suggest updating the note as follows. The idea is that the user can select the end date, however it will not be taken into consideration.

Note: If you select *Time Range > Custom*, only the *Start Date* day will be applied to the widget and the hourly timestamp will be disregarded. In addition, the *End Date* is always set to the current date, regardless of your selection.

Reply:
Done